services/userservices.js

```diff
@@ -57,20 +57,20 @@ await User.destroy({where:{id: id}})
   }
 };
 
-const updateuser1 = async (data, body_data) => {
-  await users.updateOne(data.email, body_data);
+const updateuser1 = async (email, body_data) => {
+  await User.update(body_data,{where:{email:email}});
```

Comment:
if key and value are same in the object you need not to write both
await User.update(body_data,{where:{email}});